packages/create-atomic/index.js

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env node
+const path = require('path');
+const args = process.argv.slice(2);
+const {Plop, run} = require('plop');
+const argv = require('minimist')(args);
+
+Plop.launch(
+  {
+    cwd: argv.cwd,
+    configPath: path.join(__dirname, 'plopfile.js'),
+    require: argv.require,
+    completion: argv.completion,
+  },
+  (env) => run(env, undefined, true)
+);
```

Comment:
If possible, can you try with Node16 and `import`.
We're keen to move fully to ECMA Script Module (ESM), so I'd say, especially given the simplicity of the script we could do an `index.mjs` ;)